### ISSUE TITLE
noremap and <cmd> causes crash

### ISSUE THREAD

**Issue by olimorris** [kyazdani42/nvim-tree.lua#702]
A really bizarre error that has taken me a while to isolate...

* I found that for a long time, `nvim-tree` would cause Neovim to freeze whenever I tried to perform any file mods
* I installed `defaults.nvim` and found that `nvim-tree` did work fine, thus ruling out my Mac
* I then proceeded to reinstate my config and remove all of my config modules one by one
* I isolated it down to a weird keymapping issue.

If I opened `nvim-tree` with:
```lua
vim.api.nvim_set_keymap("n", "<C-b>", "<cmd>NvimTreeToggle<CR>", { noremap = true })
```
then it would freeze. But if I used:
```lua
vim.api.nvim_set_keymap("n", "<C-b>", ":NvimTreeToggle<CR>", {})
```

it works just fine.

**Comment by olimorris** [comment]
Yep, they're located at https://github.com/olimorris/dotfiles/tree/master/.config/nvim

I should have tested it with defaults.nvim. I wonder if my plugins/init.lua file is to blame.

**Comment by kyazdani42** [comment]
Hi ! sorry for the very late answer. Did you manage to fix this ?

**Comment by kyazdani42** [comment]
closing since no news, if this happens again tell me i'll reopen this.

**Comment by Inkletblot** [comment]
I was also having this same issue until just now, although in my case neovim was just outright crashing. I couldn't work out why and came across this issue. I've been having a couple of seemingly unrelated problems recently and in attempting to fix them I realised I was running a debug build of neovim. I build nightly fairly frequently, but must have messed up at some point. I rebuilt a Release version and voila, this issue and the others I was having (mainly a multi second delay on exit of neovim) were gone. Rebuilding was the only material change between me having the issue and not. I haven't verified this by reverting to the debug version as I would have to rebuild. Either way I hope this may be useful, sorry for the noise if it's not.
